docs/spec/admission.md

```diff
@@ -55,22 +55,22 @@ If and only if the Pod has the `ns.networkservicemesh.io` annotation exists, and
 
 ```yaml
 initContainers:
-       - name: nsc
-         image: ${REPO}/${INITCONTAINER}:${TAG}
-         imagePullPolicy: IfNotPresent
-         env:
-           - name: NS_NETWORKSERVICEMESH_IO
-             value: ${value of annotation}
-         resources:
-           limits:
-             networkservicemesh.io/socket: 1
+   - name: nsm-init-container
+     image: ${REPO}/${INITCONTAINER}:${TAG}
+     imagePullPolicy: IfNotPresent
+     env:
+       - name: NS_NETWORKSERVICEMESH_IO
+         value: ${value of annotation}
+     resources:
+       limits:
+         networkservicemesh.io/socket: 1
 ```
 
 The ${INITCONTAINER} is based on the SDK's `NSMClientList` which parses `${NS_NETWORKSERVICEMESH_IO}` and spawns the needed number of clients.
 
 ${REPO}, ${INITCONTAINER}, and ${TAG} are specifiable for the mutating admission webhook container, defaulting to REPO=networkservicemesh, INITCONTAINER=nsc, TAG=latest.
 
-
+The nsm-init container will be added to the beginning of the `initContainers` list of the POD. It means that other init containers on the list can do some work with a created connection.
```

Comment:
s/a created connection/network setup prepared by NSM this container/
As "merge" example above shows - there could be multiple connections prepared by this init container.